branchVal.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+local_branch=${1}
+
+valid_branch_dash="([a-z0-9\-\]+)"
```

Comment:
This regex statement should keep all logic from the existing regex, while removing the capital letter allowance.
See https://github.com/CMSgov/macpro-quickstart-serverless/blob/master/.github/workflows/deploy.yml#L23 for the existing statement.  I think the existing statement says "Must start with a letter (either upper case or lower case), then may contain any number or sequence of upper/lower case letters, numbers, or hyphens"

You want to keep all that, but remove the upper case piece